opennms-webapp-rest/src/main/java/org/opennms/web/rest/v1/TimelineRestService.java

```diff
@@ -366,6 +378,13 @@ private OnmsOutageCollection queryOutages(final UriInfo uriInfo, final int nodeI
         return onmsOutageCollection;
     }
 
+    @Autowired
+    private NodeDao m_nodeDao;
+
+    private OnmsNode queryNode(final int nodeId) {
```

Comment:
This follows the same pattern as the `queryOutages` method above, but it's very simple so `image()` could just do `m_nodeDao.get()` directly. Thoughts?

Reply:
yeah, seems a bit silly to have the indirection